src/poolman.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __BITCOIN_POOLMAN_H__
```

Comment:
Nit: Missing license header and wrong header naming string, should be ``BITCOIN_POOLMAN_H``